scripts/index.py

```diff
@@ -46,9 +46,14 @@ def index():
         'library_s': row['Library'],
         'shelfmark_s': row['Shelfmark - Current'],
         'shelfmark_txt': row['Shelfmark - Current'],
-        'tags_txt': [tag.strip() for tag in row['Tags'].split('#')],
-        'tags_ss': [tag.strip() for tag in row['Tags'].split('#')],
+        'tags_txt': [tag.strip() for tag in row['Tags'].split('#') if tag.strip() != ''],
```

Comment:
Thanks for catching this, had just noticed it myself. Just `if tag.strip()` is sufficient.